<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="main">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>842</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:/Users/user/Desktop/start_messanging.png);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="welcom">
     <widget class="QPushButton" name="welcom_startMessanging">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>291</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>start messaging</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="start">
     <widget class="QPushButton" name="start_signUp">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white ;</string>
      </property>
      <property name="text">
       <string>sign up</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="start_signIn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>230</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>sign in</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="signUp">
     <widget class="QLineEdit" name="signUp_firstName">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
 border-top-style: solid;
border-width:1px;

 }
</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>First name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUp_lastName">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Last name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUp_username">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUp_password">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signUp_confirmPassword">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Confirm password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signUp_signUp">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signUp_back">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="signUp_errBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="signIn">
     <widget class="QLineEdit" name="signIn_username">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signIn_password">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signIn_signIn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>sign in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signIn_back">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="signIn_errBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chatPage">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>281</width>
        <height>461</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>279</width>
         <height>459</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>456</y>
      <width>842</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
